### DISCUSSION TITLE
Mutiomics data analysis (matched snRNA-seq and sn-ATAC-seq)

### DISCUSSION THREAD

**Discussion by zhenzuo2** [GreenleafLab/ArchR#1114] [Questions / Documentation] [answered]
Hi,

I am trying to use addGeneIntegrationMatrix() to integrate snRNA-seq. But the data I have is mutiomics, in which each cell has been profiled with both RNA-seq and ATAC-seq. There is no need for data integration. In this case, How can I easily merge RNA-seq into an arrow file by matched cell ID? Thank you.

**Comment by rcorces** [comment, marked as answer]
See the Multiome tutorial
https://greenleaflab.github.io/ArchR_2020/Ex-Analyze-Multiome.html

**Reply by zhenzuo2** [reply]
Thank you so much!
